PublicWebServicesAPI_AND_servercommandScripts/setUserProperties.php

```diff
@@ -13,7 +13,7 @@
 
 function do_call($request) {
   
-  $url = "http://localhost:9191/rpc/api/xmlrpc";
+  $url = "http://172.18.48.1:9191/rpc/api/xmlrpc";

```

Comment:
should we be hard coding an ip address?

Reply:
Oops  -- This content should not be here